frontend/src/app/page.tsx

```diff
@@ -10,21 +10,23 @@ export default function Home() {
 
   return (
     <div>
+       <div className="flex items-center justify-center mt-6">
       {/* Banner */}
-      <div className="relative">
+      <div className="flex items-center justify-center w-40 h-40 rounded-full shadow-lg overflow-hidden">
         <Image
-          src="/bu_banner.jpg"
+          src="/logo.png"
           alt="BU Campus Banner"
-          width={1920}
-          height={400}
-          className="w-full h-auto"
+          width={200}
+          height={200}
+          className="w-auto h-auto"
           priority
         />
       </div>
+    </div>
 
       {/* Main Content */}
       <main className="container mx-auto px-4 py-8">
-        <h1 className="text-5xl font-bold text-center text-blue-600 mb-4">
+        <h1 className="text-5xl font-bold text-center text-[#088F8F] mb-4">
```

Comment:
Same with these, tailwind constant is better here always.... tailwind constants are great for "company colors" or "brand colors" like these.